troi/patch.py

```diff
@@ -15,6 +31,15 @@ def __init__(self, debug=False):
         # Dict used for local storage
         self.local_storage = {}
 
+        self.patch_args = {**default_patch_args, **args}
+        self.pipeline = self.create(self.patch_args)
+        self._set_element_patch(self.pipeline)
+
+        # Setup extensible services
+        self.services = {}
+        self.register_service(RecordingSearchByTagService())
```

Comment:
Would probably make sense to do it in LB local or other projects that expect to use it instead.

Reply:
The global tag and artists searches used by LB Radio have been converted to these services, so that yes, local versions can substitute their own version, but the main, global version also needs a search defined.